### PR TITLE
create content pages

### PR DESCRIPTION
**Pull request by RamyAlshurafa**
#### Issues this PR relates to
#45 

=======

_NOTE: REMEMBER NOT TO REVIEW UNLESS ALL ACCEPTANCE CRITERIA HAS BEEN TICKED OFF_

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist

Please make sure you ensure the following has been done before marking your PR as awaiting-review. 

- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

**Review by fadeomar — CHANGES_REQUESTED** [review]
- [ ] in Private tenant>> Debt Advice Locator >> there is a helpful link for Debt Advice Locator not exists please check other subsections 
image

  - [ ] the title in all sub routs says "the name of the sub route" Renter" not "tenant" 
image

  - [ ] the header Colors are not correct for the sections renders the default colors in "paying my bills" should be Tertiary/Main not Secondary/Main
image

  - [ ] the Go back button doesn't exist on all content pages 
image

File: client/src/pages/OrgPublic/Section/useTopics.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+const useTopics = (id, resources) => {
+  const [topics, setTopics] = useState([]);
+  const [markedTopics, setMarkedTopics] = useState([]);
+  useEffect(() => {
+    const fetchTopics = async () => {
+      const { data, error } = await Sections.getTopics({
+        sectionId: id,
+      });
+      if (error) {
+        // message.error('Something went wrong, please try again later');
```
Comment (fadeomar) [inline client/src/pages/OrgPublic/Section/useTopics.js:13]: handle the error

File: client/src/pages/OrgPublic/Section/index.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  const [sectionData, setSectionData] = useState({});
+  const { topics, toggleMark } = useTopics(id, publicOrg?.resources);
+
+  useEffect(() => {
+    const fetchSectionData = async () => {
+      const { data, error } = await Sections.getSectionById({
+        id,
+      });
+      if (error) {
+        // message.error('Something went wrong, please try again later');
```
Comment (fadeomar) [inline client/src/pages/OrgPublic/Section/index.js:27]: handle the error

**Review by fadeomar — CHANGES_REQUESTED** [review]

**Comment by RamyAlshurafa** [comment]
>  the title in all sub routs says "the name of the sub route" Renter" not "tenant"

the content provided from Hyde is tenant which i think is the correct https://docs.google.com/spreadsheets/d/1UmIaGBR5KC9SD4KbweGL4OZVqFEtF7lB/edit#gid=978263828

File: client/src/components/Cards/Tips/index.js [hunk 3]
```diff
@@ -38,30 +36,18 @@ const getColor = (index, startingColor) => {
...
+    .map((tip, index) => (
+      <S.Tip key={index} color={getColor(index, startingColor)} mb="3">
+        <Icon icon="bulb" color={getColor(index, startingColor).icon} mr="2" />
+        {typeof tip === 'string' ? (
+          <T.H3 color={getColor(index, startingColor).text}>Tip! {tip}</T.H3>
+        ) : (
+          tip
+        )}
+      </S.Tip>
+    ));
```
[the remark this answers is not in the document]
Comment (RamyAlshurafa) [reply client/src/components/Cards/Tips/index.js]: fixed

**Pull request merged**